slsim/Lenses/lens.py

```diff
@@ -1137,15 +1217,25 @@ def deflector_mass_model_lenstronomy(self, source_index=None):
 
         # TODO: replace with change_source_redshift() currently not fully working
         # self._lens_model.change_source_redshift(z_source=z_source)
-        lens_model = LensModel(
-            lens_model_list=self._lens_mass_model_list,
-            cosmo=self.cosmo,
-            z_lens=self.deflector_redshift,
-            z_source=z_source,
-            z_source_convention=self.max_redshift_source_class.redshift,
-            multi_plane=False,
-            use_jax=use_jax,
-        )
+        if self.multi_plane:
+            lens_model = LensModel(
```

Comment:
similar comment as somewhere else: can you first configure the input options in LensModel and then only have one executable line making the LensModel() instance?